src/Ast/Disjunction.php

```diff
@@ -27,15 +27,15 @@ public function __construct(array $terms = [])
     }
 
     /**
-     * @param Term $term
+     * @param Filter $term
      */
-    public function add(Term $term)
+    public function add(Filter $term)
     {
         $this->terms[] = $term;
     }
 
     /**
-     * @return Term[]
+     * @return Filter[]
      */
     public function getTerms()
```

Comment:
Assuming changing this from `Term[]` to `Filter[]` doesn't break anything, I would suggest renaming this to `getFilters()`.